eras/conway/impl/test/data/conway-genesis.json

```
@@ -3,7 +3,8 @@
     "committeeNormal": 0,
     "committeeNoConfidence": 0,
     "hardForkInitiation": 0,
-    "motionNoConfidence": 0
+    "motionNoConfidence": 0,
+    "securityRelevant": 0
```

Consistency with DRep threshold
```suggestion
    "ppSecurityGroup": 0
```